components/youtube/new-videos.js

```diff
@@ -0,0 +1,65 @@
+const axios = require('axios')
+const youtube = {
+  type: "app",
+  app: "youtube_data_api",
+}
+
+
+module.exports = {
+  name: "Youtube - New Videos",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    youtube,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+  methods: {
+    async getVideos(pageToken=null) {
+      return await axios.get('https://www.googleapis.com/youtube/v3/search', {
+        headers: {
+          'Authorization' : 'Bearer '+this.youtube.$auth.oauth_access_token
+        },
+        params: {
+          'part' : 'snippet',
+          'type' : 'video',
+          'forMine' : true,
+          'pageToken' : pageToken
+        }
+      });
+    },
+  },  
+
+  async run(event) {
+  	var videos = [];
```

Comment:
Do you happen to use ESLint / Prettier on your local machine? We're planning to add our own linting / formatting rules to the repo, but if you setup those tools with your text editor, it should automatically format the file on save. That way you'll never get inconsistent spacing like on this line.

Reply:
Moreover, we generally prefer `let` instead of `var` for variables whose value will change in the same block. I'd replace each `var` with `let` below.

Reply:
Updated var to let.